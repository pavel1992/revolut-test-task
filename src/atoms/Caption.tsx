import styled from 'styled-components';

import { colors } from '../colors';

export const Caption = styled.div`
    color: ${colors.captionWhite};
    font-size: 14px;
    @media(max-width: 768px) {
      font-size: 12px;
    }
`;
