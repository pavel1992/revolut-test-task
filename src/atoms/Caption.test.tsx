import * as React from 'react';
import renderer from 'react-test-renderer';

import { Caption } from './Caption';


test('renders Caption correctly', () => {
  const render = renderer
    .create(<Caption>111</Caption>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
