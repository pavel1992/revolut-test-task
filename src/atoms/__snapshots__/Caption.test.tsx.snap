// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Caption correctly 1`] = `
.c0 {
  color: rgba(0,0,0,0.5);
  font-size: 14px;
}

@media (max-width:768px) {
  .c0 {
    font-size: 12px;
  }
}

<div
  className="c0"
>
  111
</div>
`;
